spec/unit/rom/relation/schema_spec.rb

```diff
@@ -81,6 +81,36 @@ class Test::Posts < ROM::Relation[:memory]
     expect(schema.foreign_key(:users)).to be(schema[:author_id])
   end
 
+  it 'allows json read/write coersion' do
+    class Test::Posts < ROM::Relation[:memory]
+      schema do
+        attribute :payload, Types::Coercible::JSON
+      end
+    end
+
+    schema = Test::Posts.schema
+    json_payload = '{"foo":"bar"}'
+    hash_payload = { "foo" => "bar" }
+
+    expect(schema[:payload][hash_payload]).to eq(json_payload)
+    expect(schema[:payload].meta[:read][json_payload]).to eq(hash_payload)
+  end
+
+  it 'allows json read/write coersion using symbols' do
+    class Test::Posts < ROM::Relation[:memory]
+      schema do
+        attribute :payload, Types::Coercible::JSON(symbol_keys: true)
```

Comment:
@solnic It is actually `JSON` in code if this is what you have meant. Although, PR description stated otherwise. Or is this comment targeted at: `HashJson` and `JsonHash`?